### PR TITLE
add optional params

### PR DESCRIPTION
**Pull request by eyalpalo**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
Added information regarding mirroring parameters. (Not mandatory).

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/integrations/mirroring.md [hunk 1]
```diff
@@ -32,6 +32,15 @@ Where:
 - **isremotesyncin** determines if mirroring from the 3rd party integration to XSOAR is supported.
 - **isremotesyncout** determines if mirroring from XSOAR to the 3rd party integration is supported.
 
+
+## Optional Prameters
+A Mirroring Integration's YAML file may have the following parameters (under the configuration section) if can be used in the integration:
```
Comment (dantavori) [inline docs/integrations/mirroring.md:37]:
```suggestion
A mirroring integration may have the following optional parameters:
```

File: docs/integrations/mirroring.md [hunk 2]
```diff
@@ -32,6 +32,15 @@ Where:
 - **isremotesyncin** determines if mirroring from the 3rd party integration to XSOAR is supported.
 - **isremotesyncout** determines if mirroring from XSOAR to the 3rd party integration is supported.
 
+
+## Optional Prameters
```
Comment (dantavori) [inline docs/integrations/mirroring.md:36]:
```suggestion
## Optional Parameters
```

File: docs/integrations/mirroring.md [hunk 3]
```diff
@@ -32,6 +32,15 @@ Where:
...
+A Mirroring Integration's YAML file may have the following parameters (under the configuration section) if can be used in the integration:
+- **incidents_fetch_query** determines the query to fetch incident with.
+- **comment_tag, work_notes_tag, file_tag** available tags for marking incident entries.
+- **mirror_direction** determines which mirroring directions are available (options are None, Incoming, Outgoing and 'Incoming and Outgoing').
+- **close_incident** checkbox determines if mirrored XSOAR incidents will be closed when the corresponding incident is closed.
+-  **close_out** checkbox determines if mirrored incidents will be closed when the corresponding XSOAR incident is closed.
```
Comment (dantavori) [inline docs/integrations/mirroring.md:42]:
```suggestion
- **close_out** checkbox determines if mirrored incidents will be closed when the corresponding XSOAR incident is closed.
```

**Comment by eyalpalo** [comment]
@jrosenpanw can someone please have a look? Thanks

**Comment by eyalpalo** [comment]
@ShirleyDenkberg  can you please have a look? Thanks

**Comment by ShirleyDenkberg** [comment]
@dantavori Doc review completed.

**Review by dantavori — APPROVED** [review]

**Pull request merged**